config/default/manager_auth_proxy_patch.yaml

```diff
@@ -1,24 +1,26 @@
-# This patch inject a sidecar container which is a HTTP proxy for the controller manager,
-# it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
+# This patch inject a sidecar container which is a HTTP proxy for the
+# controller manager, it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: spanner-autoscaler-manager
+  name: controller-manager
```

Comment:
It might be friendly to write the migration steps in the README(or another documentation) if you want to change the name.

And the same for other resources.